components/transform/transform.py

```diff
@@ -0,0 +1,138 @@
+    alpha_6 += -103.0 * math.pow(a_third_flattening, 4) / 140.0
+    alpha_6 += 15061.0 * math.pow(a_third_flattening, 5) / 26880.0
+    alpha_6 += 167603.0 * math.pow(a_third_flattening, 6) / 181440.0
+    alpha_6 += -67102379.0 * math.pow(a_third_flattening, 7) / 29030400.0
+    alpha_6 += 79682431.0 * math.pow(a_third_flattening, 8) / 79833600.0
+    coefficients.append(alpha_6)
+
+    alpha_8 = 49561.0 * math.pow(a_third_flattening, 4) / 161280.0
+    alpha_8 += -179.0 * math.pow(a_third_flattening, 5) / 168.0
+    alpha_8 += 6601661.0 * math.pow(a_third_flattening, 6) / 7257600.0
+    alpha_8 += 97445.0 * math.pow(a_third_flattening, 7) / 49896.0
+    alpha_8 += -40176129013.0 * math.pow(a_third_flattening, 8) / 7664025600.0
+    coefficients.append(alpha_8)
+
+    alpha_10 = 34729.0 * math.pow(a_third_flattening, 5) / 80640.0
+    alpha_10 += -3418889.0 * math.pow(a_third_flattening, 6) / 1995840.0
+    alpha_10 += 14644087.0 * math.pow(a_third_flattening, 7) / 9123840.0
+    alpha_10 += 2605413599.0 * math.pow(a_third_flattening, 8) / 622702080.0
+    coefficients.append(alpha_10)
+
+    alpha_12 = 212378941.0 * math.pow(a_third_flattening, 6) / 319334400.0
+    alpha_12 = -30705481.0 * math.pow(a_third_flattening, 7) / 10378368.0
+    alpha_12 = 175214326799.0 * math.pow(a_third_flattening, 8) / 58118860800.0
+    coefficients.append(alpha_12)
+
+    alpha_14 = 1522256789 * math.pow(a_third_flattening, 7) / 1383782400.0
+    alpha_14 += -16759934899.0 * math.pow(a_third_flattening, 8) / 3113510400.0
+    coefficients.append(alpha_14)
+
+    alpha_16 = 1424729850961.0 * math.pow(a_third_flattening, 8) / 743921418240.0
+    coefficients.append(alpha_16)
+
+    return coefficients
+
+class Spheroid():
+    def __init__(self, a_semi_major_axis, a_flattening):
+        # Defined constants
+        self.m_semi_major_axis = a_semi_major_axis
+        self.m_flattening = a_flattening
+        # Derived constants
+        self.m_semi_minor_axis = self.m_semi_major_axis * (1.0 - self.m_flattening)
+        self.m_semi_major_axis_squared = math.pow(self.m_semi_major_axis, 2)
+        self.m_semi_minor_axis_squared = math.pow(self.m_semi_minor_axis, 2)
+        self.m_first_eccentricity = math.sqrt((self.m_semi_major_axis_squared - self.m_semi_minor_axis_squared) / self.m_semi_major_axis_squared)
+        self.m_first_eccentricity_squared = math.pow(self.m_first_eccentricity, 2)
+        self.m_second_eccentricity = math.sqrt((self.m_semi_major_axis_squared - self.m_semi_minor_axis_squared) / self.m_semi_minor_axis_squared)
+        self.m_second_eccentricity_squared = math.pow(self.m_second_eccentricity, 2)
+
+        self.m_third_flattening = (self.m_semi_major_axis - self.m_semi_minor_axis) / (self.m_semi_major_axis + self.m_semi_minor_axis)
+        self.m_rectifying_radius = getRectifyingRadius(self.m_semi_major_axis, self.m_third_flattening)
+        self.m_coefficients = getCoefficients(self.m_third_flattening)
```

Comment:
you don't need 83 - 85 (and by extension 3 - 67) those are for GDA2020 which we're not doing